Forge/Client/Pages/Character.razor

```diff
@@ -36,18 +44,36 @@ else
                      Theme="classic"
                      IdEpxression="(item) => item.Id.ToString()"
                      TextExpression="(item) => item.Name"
-                     @bind-Value="@_selectedTags">
+                     @bind-Value="@_selectedTags"
+                     IsDisabled="@Disabled">
             </Select2>
         </div>
         <div class="form-group">
             <label for="char_description">Description</label>
             <div style="display: grid; grid-template-columns: 50% 50%; grid-template-areas: 'left right'; grid-gap: 15px">
-                <textarea class="form-control" name="char_description" rows="20" style="grid-template: left; height: 100%" placeholder="Description" @oninput="@(args => { _character.Description = args.Value.ToString(); RenderDescription(); })" @bind="_character.Description" />
+                <textarea class="form-control" name="char_description" rows="20" style="grid-template: left; height: 100%" placeholder="Description" @oninput="@(args => { _character.Description = args.Value.ToString(); RenderDescription(); })" @bind="_character.Description" readonly="@Disabled"/>
                 <div style="grid-template: right">@((MarkupString)_descriptionMarkdown)</div>
             </div>
         </div>
     </form>
-    <button class="btn btn-outline-primary" @onclick="Save">Save</button>
+    
+    @if (string.IsNullOrEmpty(Id) == false)
+    {
+        @if (_character.IsDeleted == false)
+        {
+            <button class="btn btn-outline-primary" @onclick="Save">Save</button>
+            <button class="btn btn-outline-primary" @onclick="Delete">Delete</button>
```

Comment:
Please use "danger" red color for the delete button. This will somewhat inform the user that this action is dangerous.